upgrade.php

```diff
@@ -244,7 +244,7 @@ function getSqlServer()
     //test if mysql or mariadb is used.
     $sql = 'SELECT VERSION() as version';
     $result = $link->query($sql);
-    if (strpos($result->fetch_array['version'], 'MariaDB') === false) {
+    if (strpos($result->fetch_array()['version'], 'MariaDB') === false) {
```

Comment:
<img width="662" alt="schermata 2017-07-03 alle 22 44 58" src="https://user-images.githubusercontent.com/28512/27807508-9adee3d6-6041-11e7-9b8d-b7eb7bc7a6bc.png">
No, it doesn't run on php less than 5.4

from the php manual (http://php.net/manual/en/language.types.array.php#example-62)

> As of PHP 5.4 it is possible to array dereference the result of a function or method call directly. Before it was only possible using a temporary variable.
> 
> As of PHP 5.5 it is possible to array dereference an array literal.